benches/parse_protobuf.rs

```diff
@@ -0,0 +1,15 @@
+mod helpers;
+use helpers::*;
+
+use std::ffi::c_char;
+use brunch::Bench;
+use pg_query;
+use pg_query::bindings::{pg_query_parse, pg_query_parse_protobuf};
+
+
+brunch::benches!(
+    Bench::new("pg_query_parse_protobuf")
+        .run_seeded_with(c_seed, |query| {
+            unsafe { pg_query_parse_protobuf(query.as_ptr() as *const c_char) }
```

Comment:
It might be interesting to include the deserialization in Rust as well, so we can get a sense for where to optimize (if we were to optimize the serialization itself).

For the protobuf parse benchmark we can use the same mechanism the crate currently uses. For JSON we could (just for testing) use the mechanism that [pg_parse](https://github.com/paupino/pg_parse) uses (which shares a common history with this crate, but we since diverged to focus on the Protobuf format).